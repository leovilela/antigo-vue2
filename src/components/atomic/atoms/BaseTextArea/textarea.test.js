import { shallowMount } from "@vue/test-utils";
import TextArea from "../BaseTextArea/textarea";

const factory = () => shallowMount(TextArea, {});

describe("TextArea", () => {
  test("mounts properly", () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
