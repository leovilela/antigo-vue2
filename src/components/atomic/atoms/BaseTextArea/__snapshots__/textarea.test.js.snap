// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea renders properly 1`] = `<textarea rows="10" cols="50"></textarea>`;
